docs/context/ref/ChatRoom.md

```diff
@@ -0,0 +1,66 @@
+---
+id: ChatRoom
+sidebar_label: ChatRoom
+title: Contact
+hide_title: true
+---
+# `ChatRoom`
+
+Reference to a chat room.
+
+## Type
+
+`fdc3.chat.room`
+
+## Schema
+
+https://fdc3.finos.org/schemas/next/chatRoom.schema.json
+
+## Details
+
+| Property    | Type    | Required | Example Value     |
+|-------------|---------|----------|-------------------|
+| `type`      | string  | Yes      | `'fdc3.chat.room'`  |
+| `providerName`      | string  | Yes       | `'Symphony'`      |
+| `id`  | object  | Yes       | `{ roomId: 'j75xqXy25NBOdacUI3FNBH', anyOtherKey: 'abcdef'}` |
+| `uri` | string  | No       | `'http://symphony.com/ref/room/j75xqXy25NBOdacUI3FNBH___pqSsuJRdA'`      |
+| `name` | string  | No       | `'My new room'`      |
+
+The `uri` is an universal url to access to the room. It could be opened from a browser, a mobile app, etc...
```

Comment:
```suggestion
The `uri` field should be populated with a URL to access the room, which may be used outside of the context of a Desktop Agent (for example in a browser, mobile application etc.).
```

Minor comment: is it expected to be a URL or URI? All URLs are URIs but not all URIs are URLs (although I don't know many people that are clear on the distinction). If its always expected to be retreivable in a browser (taking you to a web page where you can access teh item) rather than just acting as an identity, then its a URL (and should be named as such).